onnxruntime/core/graph/graph.cc

```diff
@@ -1231,6 +1231,26 @@ Graph::Graph(const Model& owning_model,
   ArgNameToTypeMap name_to_type_map;
   const auto& model_path = ModelPath();
 
+  // If the tensor proto data is large enough, externalize it and replace with a tensor_proto
+  // with external data reference pointing to an OrtValue, otherwise do nothing.
+  auto put_data_maybe_in_memory = [this, &model_path](ONNX_NAMESPACE::TensorProto& tensor_proto) {
```

Comment:
nit: can you clarify what this is doing? it seems to only process tensors `> utils::kSmallTensorExternalDataThreshold` (the opposite of what I would expect to be 'put in memory' and I don't see anything in `utils::TensorProtoToOrtValue` that moves something to being external so the 'externalize' aspect is unclear.